pages/confirmation/index.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="mx-auto mt-20 mb-96 w-[540px]">
+    <h1 class="mb-10 text-center">Nice, you created a booking!</h1>
```

Comment:
I'm not sure that using `h1` here is semantically good, but just to be mindful when it will actually matter in real project 😄

Reply:
@WawrzenskiPatryk This is the page, `Confirmation -> Index`, so it's ok that he's using `h1` here.